### ISSUE TITLE
add mapM_/forM_?

### ISSUE THREAD

**Issue by mitchellwrosen** [Gabriel439/Haskell-Foldl-Library#49]
Hi, do you think `mapM_` would be a useful addition to `foldl`?

```
mapM_ :: Monad m => (a -> m b) -> FoldM m a ()
mapM_ action = sink (void . action)
```

It took me a while to find `sink` as the name is unfamiliar :)

**Comment by Gabriel439** [comment]
Yeah, that sounds reasonable.  Could you submit a pull request?

**Comment by mitchellwrosen** [comment]
Sure, #50

**Comment by Gabriel439** [comment]
Fixed by #50
